### PR TITLE
Update VS Code 1.2.0 summary

### PR DESCRIPTION
**Pull request by Shanedell**
Quick fix to make the 1.2.0 summary not be the same as 1.1.0's

### CODE REVIEW

File: site/_vscode/1.2.0.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ apache: true
 title: 1.2.0
 date: 2022-12-06
 summary: >
-    Initial omega-edit client & DFDL language
+    Bug fixes and improvements
```
Comment (stevedlawrence) [inline site/_vscode/1.2.0.md:8]: Is there anything more specific than this? This could describe any release.

Comment (Shanedell) [reply site/_vscode/1.2.0.md:8]: @stevedlawrence would something like:

```text
Testing, omega-edit, intellisense, configuration and debugging improvements. Along with package upgrades and bug fixes.
```

be better?

Comment (stevedlawrence) [reply site/_vscode/1.2.0.md:8]: Sounds fine. Note you also don't need two +1's to merge to the site repo.

Comment (Shanedell) [reply site/_vscode/1.2.0.md:8]: @stevedlawrence Oh I assumed it was the same as the other repos, is it just one +1 for this one?

Comment (stevedlawrence) [reply site/_vscode/1.2.0.md]: I think in general we've just left if up to who ever opens the PR to determine the need for review and how much time to give review. It's nice to give time for other +1's, but they aren't mandatory for the site repo. And for small changes it's not required.

**Pull request merged**